### ISSUE TITLE
BR OOM when backup with version 4.0.9

### ISSUE THREAD

**Issue by shuijing198799** [pingcap/br#704]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.
br use nearly 5G memory in backup. total data size : 800G

2. What did you expect to see?
br use limited memory

3. What did you see instead?
br use nearly 5G memory in backup. total data size : 800G

4. What version of BR and TiDB/TiKV/PD are you using?
br -V
```
Release Version: v4.0.9
Git Commit Hash: 804aa908719748888010edaa944757b82f99f4df
Git Branch: heads/refs/tags/v4.0.9
Go Version: go1.13
UTC Build Time: 2020-12-19 04:53:25
Race Enabled: false
```

tidb-server -V
```
Release Version: v4.0.9
Edition: Community
Git Commit Hash: 69f05ea55e8409152a7721b2dd8822af011355ea
Git Branch: heads/refs/tags/v4.0.9
UTC Build Time: 2020-12-21 04:26:49
GoVersion: go1.13
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false
```
tikv-server -V
```
TiKV 
Release Version:   4.0.9
Edition:           Community
Git Commit Hash:   18dec72b12eafdc40a463eee8f6c32594ee4a9ff
Git Commit Branch: heads/refs/tags/v4.0.9
UTC Build Time:    2020-12-21 02:52:25
Rust Version:      rustc 1.42.0-nightly (0de96d37f 2019-12-19)
Enable Features:   jemalloc mem-profiling portable sse protobuf-codec
Profile:           dist_release
```
pd-server -V
```
Digest: sha256:e13b2a438f7bf0cc7b287463d07e60ff58801eb4d5dbb1e862e3a0c302bf8437
Status: Downloaded newer image for pingcap/pd:v4.0.9
```

The user has tens of thousands of tables. When br is counting, the backup statistics are turned on.
```
backup full --pd=xxxx:2379 --ca=/var/lib/cluster-client-tls/ca.crt --cert=/var/lib/cluster-client-tls/tls.crt --key=/var/lib/cluster-client-tls/tls.key --send-credentials-to-tikv=true --storage=xxxx --concurrency=16
```
but it will still OOM when use ignore-stats
```
backup full --pd=xxxx:2379 --ca=/var/lib/cluster-client-tls/ca.crt --cert=/var/lib/cluster-client-tls/tls.crt --key=/var/lib/cluster-client-tls/tls.key --send-credentials-to-tikv=true --storage=xxxx --concurrency=16 --ignore-stats=true
```
It seems 4.0.8 will not report this error.

**Comment by shuijing198799** [comment]
https://github.com/pingcap/br/issues/691

**Comment by tennix** [comment]
related #693

**Comment by shuijing198799** [comment]
The root cause of this issue is backup statistic data use huge mount of memory. We decide temporary disable backup statistics data to solve this issue.

**Comment by jebter** [comment]
Has it been resolved, or is it bypassed by changing to 4.0.8? @shuijing198799 @cyliu0

**Comment by cyliu0** [comment]
> Has it been resolved, or is it bypassed by changing to 4.0.8? @shuijing198799 @cyliu0

There is a workaround https://github.com/pingcap/br/pull/693 . And it has been verified in 4.0.10 by the developers. There is an internal test record if you're interested.
